Source/Diagnostics/FlushFormats/FlushFormatCheckpoint.cpp

```diff
@@ -178,8 +178,8 @@ FlushFormatCheckpoint::CheckpointParticles (
         Vector<std::string> real_names;
         Vector<std::string> int_names;
 
+        // note: positions skipped here, since we reconstruct a plotfile SoA from them
```

Comment:
@atmyers Here, I tried both adding positions (see openPMD output) and not adding. In both cases, it gets a bit confused and either misses attribute names or names them twice. Thus, I think there is just an offset issue how we iterate this name list somewhere.